docs/docs/ref-01-top-level-api.md

```diff
@@ -36,6 +36,19 @@ Create and return a new `ReactElement` of the given type. The type argument can
 html tag name string (eg. 'div', 'span', etc), or a `ReactClass` (created via `React.createClass`).
 
 
+### React.cloneElement
+
+```
+ReactElement cloneElement(
+  ReactElement element,
+  [object props],
+  [children ...]
```

Comment:
Doesn't have a type, it's var args and can be an array, an element, a string, a number, a bool... which basically maps to ReactNode but that's not really a super well defined type, so meh. We already have it exactly like this for `createElement`